pom.xml

```diff
@@ -82,22 +82,22 @@
 
     <distributionManagement>
         <repository>
-            <id>release-repository</id>
-            <url>https://oss.sonatype.org/service/local/staging/deploy/maven2</url>
+            <id>cxp-releases</id>
+            <name>CxP Release Repository</name>
+            <url>https://ncr.jfrog.io/artifactory/fsg-ess-maven-releases</url>
         </repository>
         <snapshotRepository>
-            <id>snapshot-repository</id>
-            <name>Maven2 Snapshot Repository</name>
-            <url>https://oss.sonatype.org/content/repositories/snapshots</url>
-            <uniqueVersion>false</uniqueVersion>
+            <id>cxp-snapshots</id>
+            <name>CxP Snapshot Repository</name>
+            <url>https://ncr.jfrog.io/artifactory/fsg-ess-maven-snapshots</url>
         </snapshotRepository>
     </distributionManagement>
 
     <repositories>
         <repository>
-            <id>snapshot-repository</id>
-            <name>Maven2 Snapshot Repository</name>
-            <url>https://oss.sonatype.org/content/repositories/snapshots</url>
+            <id>cxp-snapshots</id>
+            <name>CxP Snapshot Repository</name>
+            <url>https://ncr.jfrog.io/artifactory/fsg-ess-maven-snapshots</url>
```

Comment:
Are we adding the release to the snapshot?

Reply:
No actually.. i'll remove that and keep only release repository